openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -380,6 +381,19 @@ def has_ended(self):
         """
         return course_metadata_utils.has_course_ended(self.end)
 
+    def has_marketing_url(self):
+        """
+        Returns whether the course has marketing url.
+        """
+        return settings.FEATURES.get('ENABLE_MKTG_SITE') and self.marketing_url is not None
+
+    def has_social_sharing_url(self):
+        """
+        Returns whether the course has social sharing url.
+        """
+        is_social_sharing_enabled = getattr(settings, 'SOCIAL_SHARING_SETTINGS', {}).get('CUSTOM_COURSE_URLS')
+        return is_social_sharing_enabled and self.social_sharing_url is not None
```

Comment:
@Qubad786 We need to change the inline condition for both `has_marketing_url` and `has_social_sharing_url`.  Please see the discussion in hipchat room.